optimum/intel/openvino/configuration.py

```diff
@@ -350,17 +350,24 @@ class OVQuantizationMethod(str, Enum):
         "sym": False,
         "group_size": -1,
     },
-    "inceptionai/jais-13b": {
-        "bits": 4,
-        "sym": False,
-        "group_size": 128,
-        "ratio": 1.0,
-        "advanced_parameters": nncf.AdvancedCompressionParameters(
-            group_size_fallback_mode=nncf.GroupSizeFallbackMode.ADJUST,
-        ),
-    },
 }
 
+if is_nncf_available():
```

Comment:
shall we check NNCF version? When nncf.GroupSizeFallbackMode.ADJUST is introduced?

Reply:
It was introduced in NNCF 2.18, this is also the minimal requirement for NNCF https://github.com/huggingface/optimum-intel/blob/main/setup.py#L67